content/authorization.md

```diff
@@ -1176,11 +1176,6 @@ Disable or rotate your system account via Cerner Central as
   soon as possible via
   [Cerner Central System Account Management][SYSTEM-ACCOUNTS].  
 
-**Is offline_access supported for healthcare providers?**
```

Comment:
This may have been discussed somewhere already (though I didn't see it on the JIRA), but is there value in keeping this section but updating it to say that it is supported? Thinking of someone who might have previously read this FAQ and wondering if this should be updated to something like "Cerner now supports offline_access for providers" with maybe additional useful information ("as of <version or date").

Reply:
After some offline discussion it sounds like removing this item is the desired course of action.